.github/workflows/main.yml

```
@@ -10,20 +10,20 @@ jobs:
 
-    - name: Set up Docker Buildx
-      uses: docker/setup-buildx-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@f95db51fddba0c2d1ec667646a06c2ce06100226  # v1.6.0  # v1.6.0
 
-    - name: Log in to Docker Hub
-      uses: docker/login-action@v1
-      with:
+      - name: Log in to Docker Hub
+        uses: docker/login-action@0d4c9c5f114e0051d914bca15554477dd762a938  # v1.14.1  # v1.14.1
+        with:
```

actionlint (1.7.12)

[error] "with" section should not be empty. please remove this section if it's unnecessary

(syntax-check)